src/pages/style-guide.mdx

```diff
@@ -0,0 +1,162 @@
+import CondaCSVG from '@site/static/img/conda_c.svg';
+import CondaLogoSVG from '@site/static/img/conda_logo.svg';
+
+
+# Style Guide
+
+This page serves as a style guide for conda related projects.
+It contains important logos and fonts to use when creating your own conda
+branded content or websites.
+
+---
+
+## Typography
+
+Below are various typography elements. The header font for [conda.org](https://conda.org) is 
+[Nunito](https://fonts.google.com/specimen/Nunito) and the body font is 
+[Open Sans](https://fonts.google.com/specimen/Open+Sans).
+
+# Header one (2rem)
+
+## Header two (1.5rem)
+
+### Header three (1.25rem)
+
+#### Header four (1rem)
+
+##### Header five (0.875rem)
+
+###### Header six (0.85rem)
+
+Paragraph text. (1rem)
+
+---
+
+## Color Palette
```

Comment:
Has this been tested for contrast/accessibility?

https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima has a widget to give results dynamically.